mastering-plone/dexterity_2.rst

```diff
@@ -175,13 +190,16 @@ Create :file:`upgrades.zcml`:
 
     </configure>
 
-The upgrade step bumps the version number of the GenericSetup profile of :py:mod:`ploneconf.site` from 1000 to 1001. The version is stored in :file:`profiles/default/metadata.xml`. Change it to
+The upgrade step bumps the version number of the GenericSetup profile of :py:mod:`ploneconf.site` from 1000 to 1001.
+The version is stored in :file:`profiles/default/metadata.xml`.
+Change it to
 
 ..  code-block:: xml
```

Comment:
Do all instances of `code-block` render correctly? If not, remove the extra space between `..` and `code-block`, leaving just one space.

Reply:
they do, see https://training.plone.org/5/mastering-plone/behaviors_2.html